### PR TITLE
Add tips for commands which can be clicked via GUI

### PR DESCRIPTION
**Pull request by ljy0422**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -247,10 +246,12 @@ Example: `login /u johndoe /p 123456`<br>
 * A valid password is between 6 and 20 characters long and does not contain spaces. <br><br>
 * A valid username should be alphanumeric and between 4 and 10 characters long. <br><br>
   </box>
-
-**Confirmation of Successful Login**<br>
+* **Confirmation of Successful Login**<br>
 
 If the user Logins successfully, a graphical user interface (GUI) indicative of a successful login will be displayed, as illustrated below.<br>
+After you log in successfully, you can see the current addressbook file you are accessing in the status bar at the bottom of the application.
```
Comment (ShanylOng) [inline docs/UserGuide.md:261]: Adds clarity to the Login command. LGTM.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -537,17 +546,30 @@ GUI upon successful addition command <br><br>
...
 
 ### Toggling the display: `$`
 
 Toggles the display to view / hide the contacts panel of Hi:Re. This allows users to maintain a cleaner and less-cluttered application workspace, especially if there are many contacts.<br><br>
 
   Format: <code>$</code><br>
+<div markdown="block" class="alert alert-primary">
+
+**:bulb: Tip**<br>
+You can also use this command by clicking in the `View` section of the menu bar.
```
Comment (ShanylOng) [inline docs/UserGuide.md:594]: Indicates another way to toggle the display. LGTM.

**Review by ShanylOng — COMMENTED** [review]
LGTM, thanks Jiayao!

**Comment by ChuaZenKhoon** [comment]
lgtm!

**Pull request merged**